torch/_C/__init__.pyi.in

```diff
@@ -1331,8 +1331,7 @@ def autocast_increment_nesting() -> _int: ...
 def autocast_decrement_nesting() -> _int: ...
 def is_autocast_cache_enabled() -> _bool: ...
 def set_autocast_cache_enabled(enabled: _bool) -> None: ...
-def _increment_version(tensor: Tensor) -> None: ...
-def _increment_versions(tensors: List[Tensor]) -> None: ...
+def _increment_version(tensors: Iterable[Tensor]) -> None: ...
```

Comment:
The type is not correct here. Does "Iterate-able" exists? :p

Reply:
I think `Iterable` is actually the right type here? (not `Iterator` though)

Reply:
https://docs.python.org/3/library/typing.html#typing.Iterable

Reply:
```suggestion
def _increment_version(tensors: Union[Iterable[Tensor], Tensor]) -> None: ...
```